deploy/crds/smartgateway.infra.watch_smartgateways_crd.yaml

```diff
@@ -10,11 +10,11 @@ spec:
     plural: smartgateways
     singular: smartgateway
   scope: Namespaced
-  version: v1alpha1
+  version: v2alpha1
```

Comment:
The SAO has been looking for this v2alpha1 for two months now? https://github.com/redhat-service-assurance/service-assurance-operator/blame/master/deploy/olm-catalog/service-assurance-operator/0.1.0/service-assurance-operator.v0.1.0.clusterserviceversion.yaml#L132

I'm a bit confused how this was working I think.

Reply:
0.1.0 uses v1alpha1 and 0.2.0 uses v2alpha1.

Reply:
I think what is confusing is that you haven't been using smart-gateway 0.1.0 for the last two months. Only as of yesterday was `latest` == `0.1.0`. Prior to my merging fiasco it would have been getting built from the https://github.com/redhat-service-assurance/smart-gateway-operator.0.2.0 repo, which would have been version `0.2.0` (and thus, using that `v2alpha1` API version).